src/Service/OS2Forms/CaseSubmissionNormalizers/CaseSubmissionNormalizer.php

```diff
@@ -2,176 +2,140 @@
-
-            if (!$complaintCategory) {
-                $message = sprintf('Could not find a complaint category with name %s  and board id %s', $submissionData['klagetype'], $submissionData['naevn']);
-                throw new WebformSubmissionException($message, 401);
-            }
-
-            $normalizedArray['complaint_category'] = $complaintCategory;
-        } else {
-            $message = sprintf('Submission data does not contain a complaint category.');
-            throw new WebformSubmissionException($message);
-        }
-
-        // Documents
-        if (isset($submissionData['dokumenter']) && !empty($submissionData['dokumenter'])) {
-            $documents = [];
-            foreach ($submissionData['dokumenter'] as $documentId) {
-                try {
-                    $options = [
-                        'headers' => [
-                            'api-key: '.$this->selvbetjeningUserApiToken,
-                        ],
-                    ];
-
-                    // Get document information.
-                    $response = $this->httpClient->request(
-                        'GET',
-                        $sender.'/entity/file/'.$documentId,
-                        $options
-                    );
-
-                    $data = $response->toArray();
-                    $documentUrl = $sender.$data['uri'][0]['url'];
-
-                    // Get document.
-                    $response = $this->httpClient->request(
-                        'GET',
-                        $documentUrl,
-                        $options
-                    );
-
-                    // Create new document
-                    // Example basename: before.png
-                    // Example resulting newFileName: before-62b1aa95ef3cb.png
-                    $documentName = preg_replace('/\..*/', '', basename($documentUrl));
-                    $newFileName = $this->documentUploader->generateNewUniqueFileName($documentName);
-                    $filePath = $this->documentUploader->getFullDirectory().'/'.$newFileName;
-                    file_put_contents($filePath, $response->getContent());
-                    $document = $this->documentUploader->createDocumentFromPath($newFileName, $documentName, 'OS2Forms');
-                    $documents[] = $document;
-
-                    // Persist now, flush when case is created.
-                    $this->entityManager->persist($document);
-                } catch (\Exception $e) {
-                    throw new WebformSubmissionException($e->getMessage());
-                }
-            }
-
-            $normalizedArray['documents'] = $documents;
-        } else {
-            $normalizedArray['documents'] = null;
-        }
+        return $this->handleConfig($sender, $config, $submissionData);
+    }
 
-        return $normalizedArray;
+    protected function getConfig(): array
+    {
+        return [
+            'board' => [
+                'os2FormsKey' => 'naevn',
```

Comment:
For consistency this should be called `os2forms_key`.

Reply:
Changed.